src/utils/constants.js

```diff
@@ -1,11 +1,19 @@
+//@flow
 
 // number of sim curves to be displayed in Graph
-export const numDisplaySims = 30;  
+export const numDisplaySims = 30;
 
 // id: explicit key to list items in onChange()
 // key: use this to key into dataset
 // name: display name for visualization
-export const STATS = [
+
+export type Stat = {
```

Comment:
Not that I didn't have to create a completely separate file like in https://github.com/HopkinsIDD/covid-dashboard-app/pull/86/files#diff-dc8854f26b91c0ebbb07727882dbf479R5